actions/mouths-of-watersheds/mouths-of-watersheds.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import csv
+from netCDF4 import Dataset
+from shapely.geometry import shape
+import geojson
+from mow.helpers import *
+from mow.geo_data_grid_2d.vic import VicDataGrid
+
+parser = argparse.ArgumentParser('Output a CSV file with the lon lat of the mouth of each watershed')
+parser.add_argument('nc', help='an RVIC-netCDF file')
+parser.add_argument('geojson', help='a geoserver output file describing the watersheds')
+parser.add_argument('csv', help='the csv file to output the results to')
+args = parser.parse_args()
+nc = Dataset(args.nc, "r")
+flow_direction = VicDataGrid.from_nc_dataset(nc, "flow_direction")
+grid = VIC_direction_matrix(flow_direction.lat_step, flow_direction.lon_step) 
+
+with open(args.geojson) as f:
+    with open(args.csv, 'w', encoding='UTF8') as x:
```

Comment:
We don't have to nest (and don't want to). A `with` statement can have any number of comma-separated contexts defined in it:

```
with open(...) as a, open(...) as b:
```